indexer/validate_and_format_contacts.cpp

```diff
@@ -21,15 +21,20 @@ string ValidateAndFormat_facebook(string const & facebookPage)
 {
   if (facebookPage.empty())
     return {};
-  // Check that facebookPage contains valid username. See rules: https://www.facebook.com/help/105399436216001
-  if (strings::EndsWith(facebookPage, ".com") || strings::EndsWith(facebookPage, ".net"))
-    return {};
-  if (regex_match(facebookPage, s_fbRegex))
-  {
-    if (facebookPage.front() == '@')
+
+  if (facebookPage.front() == '@') {
+    // Validate facebookPage as username or page name
+    if (facebookPage.substr(1).find_first_of(kForbiddenFBSymbols) == string::npos)
```

Comment:
Not making additional substr copy: facebookPage.find_first_of(kForbiddenFBSymbols, 1)

Reply:
Removed `page.substr(1)`